moped-editor/src/views/projects/projectView/ProjectNotes.js

```diff
@@ -306,6 +306,7 @@ const ProjectNotes = (props) => {
                 submitNewNote={submitNewNote}
                 submitEditNote={submitEditNote}
                 cancelNoteEdit={cancelNoteEdit}
+                isStatusEditModal={isStatusEditModal}
```

Comment:
If we were using this input component in _way_ more places in the app, I could see naming this prop something like `hideTypeToggle` or `showTypeToggle` so that `NoteInputQuill` would only need to know whether or not to show the toggle (and less about the specific context like status modal or not status modal). No change needed here!